nbs/models.itransformer.ipynb

```diff
@@ -0,0 +1,933 @@
+{
```

Comment:
<ul><li>I think iTransformer is a multivariate model, so we need to use BaseMultivariate. This will have implications to the rest of the implementation.</li><li>Missing parameters: optimizer, optimizer_kwargs (see other models)</li><li>I believe enc_in, dec_in, and c_out should be equal to n_series</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/944/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
Used BaseMultivariate
Added missing parameters
Fixed shapes of enc_in, dec_in and c_out to equal the n_series